File: Numbers/fibonacci.py
# -*- coding: cp1252 -*-
# Fibonacci Sequence - Enter a number and have the
# program generate the Fibonacci sequence to that number
# or to the Nth number

n = int(raw_input('How many numbers do you need? '))
series = [1]

while len(series) < n:
    if len(series) == 1:
        series.append(1)
    else:
        series.append(series[-1] + series[-2])

print series

File: Numbers/Fibonacci.java
import java.util.Scanner; 

public class Fibonacci {
  public static void main (String [] args) {
    new Fibonacci ();
  }
  
  public Fibonacci () {
    Scanner input = new Scanner(System.in);  // Reading from System.in
    System.out.println("How many numbers?");
    int n = input.nextInt();
    
    int a = 0;
    int b = 1;
    int c = 0;
    
    for (int i = 0; i < n; i ++) {
      c = a + b;
      a = b;
      b = c;
      System.out.println(c);
    }
  }
}